test_requirements.txt

```diff
@@ -1,5 +1,14 @@
+cheroot>=6.0.0
 DBUtils
-mysql-connector>=2.1.4
-psycopg2
-PyMySQL
 pytest>=4.6.2
+
+# DB drivers.
+# mysql
+MySQL-python>=1.2.5; python_version == '2.7'
+PyMySQL>=0.9.3
+mysql-connector-python>=8.0.19
+
+# pgsql
+#psycopg; python_version == '2.7'
+psycopg2>=2.8.4
+#pgdb>=0.0.8
```

Comment:
Can we lose commented lines 12 ans 14?

Reply:
Not yet.

web.py supports them, but seems unitests for`psycopg` and `pgdb` failed. I had to disable them temporarily. After merged this PR, we should enable them for testing again and fix the failed tests.

Reply:
I think we can mark them as deprecated and drop the support for them in the subsequent release. When web.py started there all these libraries were in use. Not I guess psycogp2 is the only library that everyone uses.

Reply:
Both psycopg and pgdb can not even pass tests. There’s no need to wait for one more release to drop them.